src/commands/Deploy/test/domain.ts

```diff
@@ -0,0 +1,40 @@
+import { describe, expect, test } from '@jest/globals';
```

Comment:
This file should be domain.test.ts.
Please modify it.

Reply:
I updated code in here:
https://github.com/kintone-labs/kintone-cli/pull/147/commits/34235f076cdfdda4718694da2e3c33b88936d683

Please have a check.
Thank you!

Reply:
I updated code in here:
https://github.com/kintone-labs/kintone-cli/pull/147/commits/54544615c7513439c8b38b0b6bf7d269680d02e2

Please have a check.
Thank you!

Reply:
Confirmed!